### PR TITLE
ddl: set default value to zero value when add a not null column.

### PR DESCRIPTION
**Pull request by coocood**

### CODE REVIEW

File: ddl/ddl.go [hunk 1]
```diff
@@ -1014,6 +1014,15 @@ func (d *ddl) AddColumn(ctx context.Context, ti ast.Ident, spec *ast.AlterTableS
 		return errors.Trace(err)
 	}
 
+	if mysql.HasNotNullFlag(col.Flag) && col.DefaultValue == nil {
+		zeroValue := table.GetZeroValue(&col.ColumnInfo)
+		col.DefaultValue, err = zeroValue.ToString()
```
Comment (shenli) [inline ddl/ddl.go]: Seems mysql does not behave this way. It will not automatically add default value to column definition.

**Comment by shenli** [comment]
Maybe we should only update backfillColumnData function.
See: https://github.com/pingcap/tidb/blob/master/ddl/column.go#L324

**Comment by coocood** [comment]
@shenli PTAL

**Comment by shenli** [comment]
LGTM
@zimulala  PTAL

**Comment by zimulala** [comment]
LGTM

**Pull request merged**